velox/docs/monthly-updates/april-2023.rst

```diff
@@ -0,0 +1,75 @@
+********************
+April 2023 Update
+********************
+
+
+Documentation
+=============
+
+ * Add :doc:`documentation on UnsafeRow serialization <../develop/unsaferow>`.
```

Comment:
Absolute paths `/develop/unsaferow>` are better. When this page gets moved to the 2023 folder next year, this does not have to change.